src/TU-Berlin-1.0.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+  <license licenseId="TU-Berlin-1.0" name="Technische Universitaet Berlin License 1.0">
+    <crossRefs>
+      <crossRef>https://github.com/swh/ladspa/blob/7bf6f3799fdba70fda297c2d8fd9f526803d9680/gsm/COPYRIGHT</crossRef>
+    </crossRefs>
+    <text>
+      <copyrightText>
+        <p>Copyright 
+		      <alt match=".+" name="copyrightYears">1992, 1993, 1994</alt> by Jutta Degener and Carsten Bormann,
+	      </p>
+        <p>Technische Universitaet Berlin</p>
+      </copyrightText>
+    
+      <p>
+        Any use of this software is permitted provided that this notice is not
+        removed and that neither the authors nor the Technische Universitaet Berlin
```

Comment:
We could throw an `alt` tag around `the Technische Universitaet Berlin`, but I suggest we take this as is now for the 3.2 release and make this improvement for 3.3

Reply:
I agree